packages/developer-portal/src/components/apps/pipeline/dns/dns-settings-page.tsx

```diff
@@ -19,55 +61,99 @@ export const DnsSettingsPage: FC<{
+                      <BodyText hasGreyText>{domain?.ResourceRecord?.Name}</BodyText>
+                    </div>
+                  </FlexContainer>
+                  <CopyToClipboard
+                    text={domain?.ResourceRecord?.Name}
+                    onCopy={handleCopyCode(setCopyState, 'certName')}
+                  >
+                    <Button intent="default">{copyState.certName}</Button>
+                  </CopyToClipboard>
+                </Col>
+                <Col>
+                  <FlexContainer>
+                    <div>
+                      <Subtitle hasNoMargin>Value</Subtitle>
+                      <BodyText hasGreyText>{domain?.ResourceRecord?.Value}</BodyText>
+                    </div>
+                  </FlexContainer>
+                  <CopyToClipboard
+                    text={domain?.ResourceRecord?.Value}
+                    onCopy={handleCopyCode(setCopyState, 'certValue')}
+                  >
+                    <Button intent="default">{copyState.certValue}</Button>
+                  </CopyToClipboard>
+                </Col>
+              </Grid>
+            </CardWrap>
+          ))}
+          <Subtitle>Custom DNS Records</Subtitle>
+          <BodyText hasGreyText>
+            The following <code>CNAME</code> record needs to be added to your domain&apos;s DNS settings to point your
+            custom domain to the distribution deployed by this pipeline.
+          </BodyText>
+          <CardWrap className={elMb11}>
+            <Grid className={cx(elMb7)}>
+              <Col>
+                <FlexContainer>
+                  <div>
+                    <Subtitle hasNoMargin>Name</Subtitle>
+                    <BodyText hasGreyText>{dnsInfo.customDomain}</BodyText>
+                  </div>
+                </FlexContainer>
+                <CopyToClipboard text={dnsInfo.customDomain} onCopy={handleCopyCode(setCopyState, 'distroName')}>
+                  <Button intent="default">{copyState.distroName}</Button>
+                </CopyToClipboard>
+              </Col>
+              <Col>
+                <FlexContainer>
+                  <div>
+                    <Subtitle hasNoMargin>Value</Subtitle>
+                    <BodyText hasGreyText>{dnsInfo.cloudfrontUrl}</BodyText>
+                  </div>
+                </FlexContainer>
+                <CopyToClipboard text={dnsInfo.cloudfrontUrl} onCopy={handleCopyCode(setCopyState, 'distroValue')}>
+                  <Button intent="default">{copyState.distroValue}</Button>
+                </CopyToClipboard>
+              </Col>
+            </Grid>
+          </CardWrap>
+          <BodyText hasGreyText>
+            If you are using a <code>reapit.cloud</code> domain, we have automatically sent the above details to DevOps
+            for them action. Once they have merged the DNS changes, the certificate will be issued and the status will
+            update. The custom domain will then be available for use.
```

Comment:
Perhaps add this in a persistent notification so it's more obvious

Reply:
Updated:

![image](https://github.com/user-attachments/assets/5026573a-d4c0-4137-8abb-cf7712d77c37)